### ISSUE TITLE
Pathfinder

### ISSUE THREAD

**Issue by daavoo** [iterative/dvc.org#3128]
It would be nice to have, as an entry point to the docs, a page where users would answer a few Qs (what's your role, what type of data you use). We could then use the answers to propose links to docs we consider relevant:

E.g.:
data scientists -> Experiment Management; 
ML engineer → Fast ... Data Caching Hub; 
Computer Vision -> Large Dataset Optimization;

**Comment by iesahin** [comment]
Instead of asking questions, we can have some links, predefined entry points that we put to highlighted boxes in the home page. These may change every three seconds: 

DVC for Data Scientists
....

DVC for ML Engineers
...

DVC for MLOps Engineers
...

DVC for AI Researchers
...

**Comment by shcheklein** [comment]
Again, reminds me a multi choice thing like https://spacy.io/usage

**Comment by daavoo** [comment]
> Again, reminds me a multi choice thing like https://spacy.io/usage

https://pytorch.org/ also uses similar multi choice

**Comment by iesahin** [comment]
Related #144

**Comment by iesahin** [comment]
I remembered we've started some work similar to this in April. We have a very brief document in Notion: https://www.notion.so/iterative/wip-Audiences-ccb7abb9a198476aa0c01581479c5eaf

Determining distinct *personas* and directing them to the relevant parts of the documentation might be good, but I'm not sure asking the user questions is a right way. People usually don't like to answer questions that must be answered as a prerequisite. 

We can create pathways for different kinds of users and make these obvious (in the sidebar, in the landing page.) Suppose the sidebar looks like


* Get Started
** DVC for ML Engineers [Access, Versioning, Pipelines, Experiments]
** DVC for Data Scientists [Experiments]
** DVC for MLOps Engineers [Access, Versioning, Pipelines]
** DVC for Software Engineers [Pipelines, Versioning]
...
** Get Started for Everyone
...

instead of the current Data Versioning / Data Access ... list. A student/enthusiast may start from any of them and we'll be saved from creating too many static pathways. 

These pathways may correspond to trails in the GS and may use common content when appropriate.
